services/QuillLMS/spec/helpers/navigation_helper_spec.rb

```diff
@@ -217,7 +271,7 @@
       before { allow(teacher).to receive(:should_render_teacher_premium?).and_return(false) }
 
       it 'should return the expected tabs' do
-        expected_tabs = tabs.push(helper.premium_tab)
+        expected_tabs = tabs
```

Comment:
I recognize that this isn't your original code, so feel free to skip this, but `expected_tabs = tabs` makes those variable names refer to the same underlying value.  So after this, `tabs.push` and `expected_tabs.concat` is actually equivalent to `tabs.push.concat`.  This could be cleaned up by just removing `expected_tab` and using `tab` in these expectations.